components/director/internal/domain/runtime/resolver_test.go

```diff
@@ -1,5 +1,6 @@
 package runtime_test
 
+/*
```

Comment:
I guess it's leftover? and the test fixtures should be adapted?

Reply:
My bad, will adapt them.